### PR TITLE
FAQ Was kann ich tun, wenn ich mein Testergebnis oder Testzertifikat nicht erhalten habe?

### PR DESCRIPTION
**Pull request by dsarkar**
image

image

image

---
Internal Tracking ID: [EXPOSUREAPP-10410](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10410)

### CODE REVIEW

**Comment by Ein-Tim** [comment]
Suggestion to link this FAQ entry from coronawarn.app/en/faq/#qr_test

**Comment by dsarkar** [comment]
EN

image

image


image

**Comment by dsarkar** [comment]
@Ein-Tim thanks for the suggestion. Done!

File: src/data/faq.json [hunk 1]
```diff
@@ -65,12 +65,25 @@
...
                             "Another list of test sites from the GitHub community: <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Where to get tested?</a>"
                         ]
+                    },
+                    {
+                        "title": "What can I do if I have not received my test result or test certificate?",
+                        "anchor": "test_result_issues",
+                        "active": true,
+                        "textblock": [
+                            "Results of <b>rapid antigen tests</b> should be transmitted via the CWA within <b>half an hour</b>, results of </b>PCR tests</b> within <b>48 hours</b>.",
+                            "If you have not received your <b>test result</b>, or if the test result has been transmitted but not the <b>digital certificate of the EU</b>, you can report this or other problems as follows:",
```
Comment (MikeMcC399) [inline src/data/faq.json:82]: Please check the name used on actual test certificates:

"digital certificate of the EU" sounds like a literal translation of the German.

more likely:
"EU digital COVID test certificate"

File: src/data/faq.json [hunk 2]
```diff
@@ -65,12 +65,25 @@
...
+                            "<ul><li>Please search for the test site on  <a href='https://map.schnelltestportal.de' target='_blank' rel='noopener'>https://map.schnelltestportal.de</a> (German only).</li><li>Tap 'Fehler melden' at the bottom right (see Fig. 1).</li><li>In the drop-down field (see Fig. 2), select the category 'Testergebnis kommt nicht an'( 'Test result does not arrive') or 'Testzertifikat kommt nicht an' ('Test certificate does not arrive') (see Fig. 3).</li><li>Optionally, you have the possibility of describing your problem in the comment field (see Fig. 2).</li><li>Tap 'Abschicken' at the bottom left in order to submit your report (see Fig. 2).</li> </ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json:83]: add space before bracket
"kommt nicht an'("
=> "kommt nicht an' ("

File: src/data/faq_de.json [hunk 3]
```diff
@@ -65,12 +65,25 @@
...
                             "Weitere Liste mit Teststellen aus der GitHub-Community: <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Wo kann man sich testen lassen?</a>"
                         ]
+                    },
+                    {
+                        "title": "Was kann ich tun, wenn ich mein Testergebnis oder Testzertifikat nicht erhalten habe?",
+                        "anchor": "test_result_issues",
+                        "active": true,
+                        "textblock": [
+                            "Ergebnisse von <b>Schnelltests</b> sollten innerhalb von einer <b>halben Stunde</b>, Ergebnisse von <b>PCR-Tests</b> innerhalb von <b>48 Stunden</b>, über die CWA übermittelt werden.",
+                            "Sollten Sie Ihr <b>Testergebnis</b> nicht erhalten haben, oder zwar das Testergebnis aber kein entsprechendes <b>Digitales Zertifikat der EU</b> übermittelt worden sein, können Sie folgendermaßen dieses oder auch andere Probleme melden:",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:82]: Check official name:
"Digitales Zertifikat der EU"

"Digitales COVID-Testzertifikat der EU" ?

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One missing space, otherwise just minor comments / questions. See in-line comments.

File: src/data/faq_de.json (hunk 3 above)
Comment (dsarkar) [reply src/data/faq_de.json]: ok, thanks

**Comment by dsarkar** [comment]
image

image

@MikeMcC399 Thanks for the review.

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json]: yep, thanks

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json]: sure, usually it is just EU digital COVID certificate, however, let's use the word 'test' to make it clearer.

**Pull request merged**